### DISCUSSION TITLE
Pkl in kotlin

### DISCUSSION THREAD

**Discussion by ghost** [apple/pkl#189] [Q&A] [answered]
how do I create a pkl file in android studio, and how can I use it with kotlin?

**Comment by bioball** [comment, marked as answer]
Take a look at our docs for integrating with Kotlin: https://pkl-lang.org/main/current/kotlin-binding/index.html

You should be able to use our [pkl-intellij](https://pkl-lang.org/intellij/current/index.html) plugin in Android Studio, and any other JetBrains editor.
